packages/components/src/components/hds/app-footer/status-link.ts

```diff
@@ -51,6 +61,7 @@ export default class HdsAppFooterStatusLinkComponent extends Component {
         ).join(', ')} received: ${this.args.status}`,
         Object.keys(STATUSES).includes(status)
       );
+      return status as HdsAppFooterStatusTypes;
     }
     return status;
```

Comment:
possibly for a different time, but it seems like we don't really want to allow this component to work if no status is provided to it

Reply:
According to [the documentation](https://helios.hashicorp.design/components/app-footer?tab=code#appfooterstatuslink) you can pass `@text` instead of status?

Reply:
Good point!